cdi/src/main/java/io/smallrye/config/inject/InjectionMessages.java

```diff
@@ -11,35 +10,29 @@
 import org.jboss.logging.annotations.MessageBundle;
 import org.jboss.logging.annotations.Param;
 import org.jboss.logging.annotations.Pos;
-import org.jboss.logging.annotations.Signature;
 
 @MessageBundle(projectCode = "SRCFG", length = 5)
 public interface InjectionMessages {
     InjectionMessages msg = Messages.getBundle(InjectionMessages.class);
 
-    @Signature(messageIndex = 1, value = { String.class, String.class })
-    @Message(id = 2000, value = "No Config Value exists for required property %s")
-    ConfigException noConfigValue(@Param @Pos(1) String key);
+    @Message(id = 2000, value = "Failed to Inject @ConfigProperty for key %s since the config property could not be found in any config source")
+    ConfigInjectionException noConfigValue(@Param @Pos(1) String key);
 
-    @Signature(messageIndex = 1, value = { String.class, String.class, Throwable.class })
-    @Message(id = 2001, value = "Failed to retrieve config for key %s")
-    ConfigException retrieveConfigFailure(@Cause IllegalArgumentException cause, @Param @Pos(1) String key);
+    @Message(id = 2001, value = "Failed to Inject @ConfigProperty for key %s %s")
+    ConfigInjectionException retrieveConfigFailure(@Param @Pos(1) String key, @Param @Pos(2) String cuase, @Cause Exception e);
```

Comment:
Also just noticed a typo. I'll correct this with any other suggestions.
```suggestion
    ConfigInjectionException retrieveConfigFailure(@Param @Pos(1) String key, @Param @Pos(2) String cause, @Cause Exception e);
```